src/components/Tabs.js

```diff
@@ -205,4 +128,184 @@ class TabWithoutRadium extends React.Component {
 
 const Tab = Radium(TabWithoutRadium);
 
+class MenuTabs extends StandardTabs {
```

Comment:
i believe this would be the first time we'd be extending from a component (that isn't React's base Component). i sometimes think inheritance gets very confusing, and maybe something like composition would work in this case? [this is a little old, but has some good concepts](https://discuss.reactjs.org/t/best-practices-for-extending-subclassing-components/1820/10)